tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -5308,7 +5308,7 @@ public function findEntryFromPropfindResponse(
 		foreach ($multistatusResults as $multistatusResult) {
 			$entryPath = $multistatusResult['value'][0]['value'];
 			if ($method === "REPORT") {
-				if ($entryNameToSearch !== null && str_ends_with($entryPath, $entryNameToSearch)) {
+				if ($entryNameToSearch !== null && substr($entryPath, strrpos($entryPath, '/') + 1) === $entryNameToSearch) {
```

Comment:
https://drone.owncloud.com/owncloud/ocis/25301/51/5
The test scenarios have various combinations of expected things in the search result table. For example:
```
    And the search result of user "Brian" should contain these entries:                 # FeatureContext::thePropfindResultShouldContainEntries()
      | /SubFolder1                                |
      | /SubFolder1/subFOLDER2                     |
      | /SubFolder1/subFOLDER2/insideTheFolder.txt |
      response does not contain the entry '/SubFolder1'
      Failed asserting that a boolean is not empty.
```

It might be easier and better if the table of search results always has the "full" path (relative to whatever the root is), and this `if` statement does very little to change `$entryPath` (maybe it needs to ignore some fixed thing at the start - I don't know exactly what the whole string looks like)

Because anyway, if a use has `file.txt` and `folder/file.txt` we want to be able to specify which of those 2 files should be in the search result, and which should not. Or that both files should be in the search result.

Reply:
I have implemented another way to strictly check the file path. With this some scenario should be refactor to have exact path.